docs/package.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "docs",
+  "version": "0.0.1",
+  "scripts": {
+    "docs": "node_modules/@openzeppelin/docs-utils/oz-docs.js -c .",
```

It isn't necessary to write the full path here. Tested locally.

```suggestion
    "docs": "oz-docs -c .",
```